src/api/spec/controllers/webui/users/tokens_controller_spec.rb

```diff
@@ -95,6 +95,18 @@
         it { expect { subject }.not_to change(Token, :count) }
       end
     end
+
+    context 'type is service' do
+      context 'and the description is larger than 64 characters' do
+        render_views
+
+        let(:form_parameters) { { token: { type: 'service', description: 'Something' * 20 } } }
+
+        include_examples 'check for flashing an error'
```

Comment:
You are testing `text_field`, I'm sure the `ActionView` people do this already :)

Reply:
Hmmm, this PR should be based on top of the referenced one... I removed that test [here](https://github.com/openSUSE/open-build-service/pull/13854#pullrequestreview-1301646479)... Let me rebase it

Reply:
Done